docs/r2v/detailed_guide.md

```diff
@@ -0,0 +1,164 @@
+the 6 preset markers, shown in Figure 2, indicating the angle of tilt. While 
+the LiDAR is in the desired position (normally full tilt 20 degrees), tighten 
+the allen screws back down to secure the LiDAR.
+
+<div class="img_container">
+    <img class='lg_img' src="../imgs/real_to_virtual_hardware_close_up.png"/>
+</div>
+
+Prior to data collection, it is important to calibrate the GNSS located on or 
+inside the R2V data collection system’s column. Xsens provides a “Magnetic 
+Field Mapper” utility that comes pre-installed with the [Xsens SDK]
+(https://content.xsens.com/mt-software-suite-download). This utility allows
+calibration of the local magnetic field to achieve accurate heading estimation. 
+Please refer to the Xsens “Magnetic Field Manual” (Document MT0202P, Revision 
+M, 21 Nov 2018) section 3.3 for instructions on how to use the field mapping 
+software. NOTE: When calibrating the GNSS magnetic field on top of a vehicle, 
+you should drive in 2 consecutive circles in either direction.
+
+### Data Logging Solution
+
+After the sensor suite is connected and calibrated, the monoDrive R2V Data 
+Logger is used to collect binary stream data on a laptop computer. The data 
+logger, shown in Figure 3, is a LabView application which enables configuration 
+of sensor settings and view mission critical sensor data during a logging 
+session. After a logging session is complete, the binary sensor stream data can 
+be found in the directory selected in the data logging application. This binary 
+data includes all sensor stream data for Camera, LiDar, and GNSS.
+
+<div class="img_container">
+    <img class='wide_img' src="../imgs/data_logger_main_tab.png"/>
+</div>
+
+<div>&nbsp;</div>
+
+<div class="img_container">
+    <img class='wide_img' src="../imgs/data_logger_config_tab.png"/>
+</div>
+
+
+### Recording Data
+Prior to recording data, make sure the GNSS has been calibrated. For best 
+results, run the recording software for a few minutes while driving. It is best 
+to rotate the vehicle at least 360 degrees by turning around several corners to 
+allow the GNSS to get acclimatized to the environment. 
+
+#### Hardware Prerequisites:
+* Connect the Ladybug Camera’s USB cable to a USB 3.0 port on the recording laptop
+
+* Connect the GNSS USB cable from the R2V system’s data collection electrical box to a USB port on the recording computer.
+
+* Connect an Ethernet cable to the R2V ethernet port on the data collection computer.
+
+* On the data collection computer, make sure the Ethernet port connected to the R2V system is on a local IP of **192.168.1.200**
+
+#### Data Logger Recording:
+1. Open the **r2v.lvprj** file on the R2V folder
+
+1. Double-click on the **main.vi**
+
+1. Click on the arrow to run the main.vi
+
+1. Select the path where the logs are going to be stored using the “Directory 
+to Log” control: 
+    a. The program will create the following directory structure to store your data:
```

Comment:
Missing a new line 
![image](https://user-images.githubusercontent.com/10260895/79772863-41756600-82f6-11ea-8884-0e3ae48b3ba0.png)

Reply:
![Screenshot from 2020-04-20 12-03-27](https://user-images.githubusercontent.com/6026642/79778831-fd3a9380-82fe-11ea-9479-fdac465d2241.png)
Hmmm this is rendering correctly for me... let me know after you pull if it still doesn't look right to you.